exercises/isogram/isogram_test.rb

```diff
@@ -76,6 +76,6 @@ def test_duplicated_character_in_the_middle
 
   def test_bookkeeping
     skip
-    assert_equal 4, BookKeeping::VERSION
+    assert_equal 5, BookKeeping::VERSION
```

Comment:
Version should be the same, non-breaking change for current solutions, no change in the tests themselves.